src/main/java/frc/robot/commands/DefaultDriveCommand.java

```diff
@@ -29,12 +29,12 @@ public void execute() {
     double slowFactor = m_driver.getIsSlowMode() ? DriveConstants.kSlowDriveFactor : 1;
     double slowRotFactor = m_driver.getIsSlowMode() ? DriveConstants.kSlowRotFactor : 1;
 
-    int reversedForRed = DriverStation.getAlliance() == Alliance.Blue ? -1 : 1;
+    int reversedForRed = DriverStation.getAlliance() == Alliance.Blue ? 1 : -1;
 
     /* Drive */
     m_swerve.drive(
       m_driver.getForwardTranslation() * slowFactor * reversedForRed,
-      m_driver.getSideTranslation() * slowFactor * reversedForRed,
+      m_driver.getSideTranslation() * slowFactor * -reversedForRed,
```

Comment:
this doesn't change anything. If you want the variable name to be accurate just change it to alianceReversal